constants/Database.ts

```diff
@@ -26,7 +26,9 @@ declare global {
     var gun : IGunInstance<any>;
     var user :IGunUserInstance<any, any, any, IGunInstanceRoot<any, IGunInstance<any>>>;
     var SEA : ISEA;
+    var userPub : string;
 }
 global.gun = gun
 global.user = gun.user()
-global.SEA = SEA
\ No newline at end of file
+global.SEA = SEA
+global.userPub = ""
```

Comment:
This global is used in many places, but is always just ""?

Reply:
Set now when logging in.